ui/pages/unlock-page/unlock-page.component.js

```diff
@@ -95,7 +95,7 @@ export default class UnlockPage extends Component {
     } catch ({ message }) {
       this.failed_attempts += 1;
 
-      if (message === 'Incorrect password') {
+      if (message.startsWith('Incorrect password')) {
```

Comment:
Curious why this is necessary now

Reply:
Hm, this may not be relevant anymore but IIRC at some point things got tagged on at the end of an error message. Do you prefer it reverted to keep changes down or fine to keep as a defensive measure?

Reply:
I would prefer it reverted if it isn't needed, just to reduce the diff